core/dictgen/src/rootcling_impl.cxx

```diff
@@ -4823,7 +4823,7 @@ int RootClingMain(int argc,
    }
    if (!gOptIgnoreExistingDict && gOptNoGlobalUsingStd) {
       if (dictStream) {
```

Comment:
check ``if (dictStream)`` made sense only when the ``dictStream`` was a pointer...

Reply:
Good point, I will fix that.